docs/getting-started/core-concepts/clusters.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Clusters"
+weight: 3
+---
+
+## What are Clusters?
+
+In the context of Greenhouse a Cluster represents a Kubernetes cluster that is onboarded to Greenhouse. Onboarded in this context means that Greenhouse can handle the management of role-based access control ([RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)) and the provisioning of operating tools (e.g. logging, monitoring, ingress etc.).
```

Comment:
I love the metaphor “on board” because it's like a ship and Greenhouse is the captain. That also makes sense because Kubernetes is derived from the Greek “captain”.